MIGRATIONS.md

```diff
@@ -18,3 +18,9 @@ Migration details:
 
 - Only `Account` objects where `isWpcom == YES` will be migrated, added a predicate filter to the mapping model
 - `Blog` has a custom migration policy to calculate `isHostedAtWPcom` and `username`
+
+## WordPress 35 (@sergioestevao 2015-07-08)
```

Comment:
I think it's a common practice to do changelogs with the newer versions at the top. Also, and I know it's most likely not your fault, where is 34?